examples/tree_2d_dgsem/elixir_euleracoustics_co-rotating_vortex_pair.jl

```diff
@@ -0,0 +1,347 @@
+using LinearAlgebra: norm
```

Comment:
Attached to this file only for convenience:

Please also create a `NEWS.md` entry.

Reply:
Done in f72bc5fe.